src/odemis/dataio/test/tiff_test.py

```diff
@@ -2387,7 +2387,49 @@ def testConvertTFStoOdemisMD(self):
             else:
                 self.assertEqual(value, data.content[0].metadata[key])
 
+    def testConvertZEISStoOdemisMD(self):
```

Comment:
This lower and uppercase mix makes the function hard to read. I'd suggest: `testConvertZeissToOdemisMetadata`

Alternatively you could go all_lower_case, as we have decided for the new functions.